<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monitor</class>
 <widget class="QWidget" name="Monitor">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Monitor{
background-color: rgb(245, 245, 220);
background-color: rgb(255, 255, 255);
}
QLabel{
font: 18pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(80, 175, 185);
color: rgb(0, 0, 0);
}
QDialog{
font: 18pt &quot;文泉驿等宽微米黑&quot;;
background-color: rgb(255, 255, 255);
}
QPushButton{
width:80px;
height:35px;
font: 20pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(0, 0, 0);
border:2px solid rgb(0, 122, 165);
border-radius:10px;
}
</string>
  </property>
  <widget class="QLabel" name="lbTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿等宽微米黑</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(255, 255, 255);
background-color:rgb(0, 122, 165);</string>
   </property>
   <property name="text">
    <string> 监控状态</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWgtPass">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>850</width>
     <height>560</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
color: rgb(255, 255, 255);
background-color: rgb(15, 41, 68);
background-color: rgb(245, 245, 220);
background-color: rgb(255, 255, 255);
}
QLCDNumber{
font: 14pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(80, 175, 185);
border:1px solid white;
border-color: rgb(79, 229, 219);
border-radius:8px;
}

QToolButton{
font: 14pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(255, 255, 255);
border:2px solid rgb(28, 183, 200);
border-radius:10px;
background-color: rgb(43, 79, 102);
background-color: rgb(15, 41, 68);
}
QToolButton:disabled{
font: 18pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
}
QLabel{
font: 14pt &quot;文泉驿等宽微米黑&quot;;
font: 16pt &quot;Noto Sans [monotype]&quot;;
color: rgb(80, 175, 185);
color: rgb(0, 0, 0);
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>38</y>
     <width>800</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;文泉驿等宽微米黑&quot;;
background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(28, 183, 200, 255), stop:1 rgba(0, 85, 127, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbSystemTime">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>5</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿微米黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;文泉驿微米黑&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>2019年01月31日 13:23:40</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>40</y>
     <width>3</width>
     <height>560</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>855</x>
     <y>40</y>
     <width>168</width>
     <height>560</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
font: 15pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(0, 101, 150);
border:2px solid rgb(0, 122, 165);
border-radius:10px;
background-color: rgb(255, 255, 255);
}
QToolButton:disabled{
font: 15pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(180, 180, 180);
border-color: rgb(180, 180, 180);
background-color: rgb(255, 255, 255);
}
QLabel{
font: 16pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(80, 175, 185);
color: rgb(0, 0, 0);
}</string>
   </property>
   <widget class="QLabel" name="lbNormalTotal">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>70</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbErrorTotal">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>125</y>
      <width>70</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbTotalPass">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>15</y>
      <width>41</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font: 14pt &quot;文泉驿等宽微米黑&quot;;
color:rgb(0, 122, 165);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>125</y>
      <width>150</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;文泉驿等宽微米黑&quot;;
border:1px solid;
border-radius:5px;
background-color: rgb(255, 233, 32);
</string>
    </property>
    <property name="text">
     <string>故障</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbOffLineTotal">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>215</y>
      <width>70</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtnMute">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>150</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>静音按钮</string>
    </property>
    <property name="icon">
     <iconset resource="../image.qrc">
      <normaloff>:/Image/speaker.png</normaloff>:/Image/speaker.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>106</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font: 14pt &quot;文泉驿等宽微米黑&quot;;
color:rgb(0, 122, 165);
}</string>
    </property>
    <property name="text">
     <string>回路总数</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbAlarmTotal">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>70</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>215</y>
      <width>25</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>个</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtnBackHome">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>485</y>
      <width>150</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>返回按钮</string>
    </property>
    <property name="icon">
     <iconset resource="../image.qrc">
      <normaloff>:/Image/back.png</normaloff>:/Image/back.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>150</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;文泉驿等宽微米黑&quot;;
border:1px solid;
border-radius:5px;
background-color: rgb(71, 167, 96);
background-color: rgb(14, 221, 111);

</string>
    </property>
    <property name="text">
     <string>正常</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>15</y>
      <width>25</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font: 14pt &quot;文泉驿等宽微米黑&quot;;
color:rgb(0, 122, 165);
}</string>
    </property>
    <property name="text">
     <string>条</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtnReset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>335</y>
      <width>150</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>复位按钮</string>
    </property>
    <property name="icon">
     <iconset resource="../image.qrc">
      <normaloff>:/Image/reset.png</normaloff>:/Image/reset.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbAlarm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>150</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;文泉驿等宽微米黑&quot;;
border:1px solid;
border-radius:5px;
background-color: rgb(255, 0, 0);
background-color: rgb(255, 77, 70);</string>
    </property>
    <property name="text">
     <string>报警</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>170</y>
      <width>25</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>个</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>125</y>
      <width>25</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>个</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtnLoopSwitch">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>150</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>回路-1</string>
    </property>
    <property name="icon">
     <iconset resource="../image.qrc">
      <normaloff>:/Image/arrow-down.png</normaloff>:/Image/arrow-down.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>215</y>
      <width>150</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;文泉驿等宽微米黑&quot;;
border:1px solid;
border-radius:5px;
background-color: rgb(171, 171, 171);</string>
    </property>
    <property name="text">
     <string>离线</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>80</y>
      <width>25</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>个</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>168</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;文泉驿等宽微米黑&quot;;
background-color:rgb(0, 122, 165);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbPowerLost">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>150</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;文泉驿等宽微米黑&quot;;
border:1px solid;
border-radius:5px;
background-color: rgb(255, 0, 0);
background-color: rgb(255, 77, 70);</string>
    </property>
    <property name="text">
     <string>电源</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbPowerLostTotal">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>70</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <zorder>label_20</zorder>
   <zorder>label_16</zorder>
   <zorder>label_18</zorder>
   <zorder>label_17</zorder>
   <zorder>lbAlarm</zorder>
   <zorder>lbPowerLost</zorder>
   <zorder>label_9</zorder>
   <zorder>label_14</zorder>
   <zorder>label_12</zorder>
   <zorder>lbNormalTotal</zorder>
   <zorder>lbErrorTotal</zorder>
   <zorder>lbTotalPass</zorder>
   <zorder>lbOffLineTotal</zorder>
   <zorder>tBtnMute</zorder>
   <zorder>label_24</zorder>
   <zorder>tBtnBackHome</zorder>
   <zorder>label_25</zorder>
   <zorder>tBtnReset</zorder>
   <zorder>tBtnLoopSwitch</zorder>
   <zorder>label_21</zorder>
   <zorder>lbAlarmTotal</zorder>
   <zorder>lbPowerLostTotal</zorder>
  </widget>
  <zorder>stackedWgtPass</zorder>
  <zorder>label_19</zorder>
  <zorder>lbTitle</zorder>
  <zorder>lbSystemTime</zorder>
  <zorder>line</zorder>
  <zorder>widget</zorder>
 </widget>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
